src/styles/style.css

```diff
@@ -0,0 +1,293 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&family=Roboto+Slab:wght@200;400;600&display=swap");
+
+body {
+  padding: 0%;
+  margin: 0%;
+  width: 100%;
+  font-family: Montserrat, sans-serif;
+  height: auto;
+  background-color: #f5f6fa;
+}
+
+h1 {
+  font-family:
+    Montserrat,
+    "Lucida Sans",
+    "Lucida Sans Regular",
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Geneva,
+    Verdana,
+    sans-serif;
+  color: #0290ff;
+  text-align: center;
+  font-weight: 600;
+  font-size: 25px;
+}
+
+.panel-bg {
+  margin: 7% 7.5%;
+  width: 85vw;
+  height: auto;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: white;
+}
+
+.navbar {
+  display: flex;
+  height: 10vh;
+  align-items: center;
+  justify-content: space-between;
+  padding: 2% 10%;
+  border-bottom: 1px solid #d2eaf1;
+}
+
+.nav-brand {
+  display: flex;
+  align-items: center;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  padding: 0%;
+}
+
+.nav-item {
+  text-decoration: none;
+  color: #121212;
+  margin: 0% 50%;
+}
+
+.nav-item:hover {
+  color: #0290ff;
+}
+
+.user {
+  width: 4vh;
+  height: 4vh;
+  border: 1px solid #0290ff;
+  border-radius: 50px;
+  color: #0290ff;
+  padding: 6px;
+}
+
+.book-list {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  padding: 0% 10%;
+}
+
+.book-panel {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 15px 0;
+  padding: 10px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+.book-specs {
+  align-self: center;
+}
+
+.category {
+  margin: 0;
+  color: #6e6e6e;
+  font-weight: 600;
+}
+
+.title {
+  margin: 0;
+  color: #121212;
+  font-family: "Roboto Slab", sans-serif;
+  font-weight: 600;
+  font-size: 30px;
+  letter-spacing: 0.4px;
+}
+
+.author {
+  margin: 0;
+  font-family:
+    "Roboto Slab",
```

Comment:
[OPTIONAL] Maybe we could make a very selective class for the `Robot Slab` being used in very specific places of our design, and then we could just apply this class to each component where is need it. I think this could be a good approach to avoid repetition.

Reply:
Good point I will keep that in mind, thanks for sharing